Errors while uninstalling ARK Core

Only Ark Core was installed.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to A, then P to uninstall Ark 
2. See error(s)

**Expected behavior**
Don't attempt uninstalling Forger if it was not installed
`    ==> Deleting Forger...
[PM2][ERROR] Process ark-core-forger not found`

Same for Relay;
`    ==> Deleting Relay...
[PM2][ERROR] Process ark-core-relay not found`

Why does DB access need sudo if we know the credentials?
`    ==> Destroying Database...
[sudo] password for ark:
    ==> Destroyed Database!
    ==> Dropping Database User...
    ==> Dropped Database User!
    ==> Uninstalled ARK Core!`

Additional errors:
`    Press any key to continuepath.js:1087
          cwd = process.cwd();
                        ^

Error: ENOENT: no such file or directory, uv_cwd
    at Object.resolve (path.js:1087:25)
    at Function.Module._resolveLookupPaths (internal/modules/cjs/loader.js:435:17)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:542:20)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at /home/ark/.nvm/versions/node/v9.11.1/lib/node_modules/npm/bin/npm-cli.js:19:21
    at Object.<anonymous> (/home/ark/.nvm/versions/node/v9.11.1/lib/node_modules/npm/bin/npm-cli.js:92:3)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
sh: 0: getcwd() failed: No such file or directory
path.js:1087
          cwd = process.cwd();
                        ^

Error: ENOENT: no such file or directory, uv_cwd
    at Object.resolve (path.js:1087:25)
    at Function.Module._resolveLookupPaths (internal/modules/cjs/loader.js:435:17)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:542:20)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/usr/share/yarn/bin/yarn.js:25:13)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
could not identify current directory: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory


**Server (please complete the following information):**
 - OS: Ubuntu 16.04

--- a/actions/core/uninstall.sh
+++ b/actions/core/uninstall.sh
@@ -14,6 +14,9 @@ core_uninstall ()
 
     database_drop_user
 
+    # Ensure we are not in a directory we are going to delete
+    cd "$commander_dir"
+
     sudo rm -rf "$CORE_DIR"
     sudo rm -rf "$CORE_DATA"
 

